lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -213,8 +213,17 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   // 1) Materials
 
   // Build the unique share link.
-  $share_path = 'user/' . dosomething_user_get_field('field_northstar_id');
-  $custom_social_share_link = dosomething_global_url(current_path(), array('absolute' => TRUE,'query' => array('source' => $share_path)));
+  $utm_source = 'influencer';
```

Comment:
all this stuffs already can already be set with a [function](https://github.com/DoSomething/phoenix/blob/a810fa26d6fc9051d1b688e0f7c6bd448087abe9/lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module#L577), ya?